src/name_bazaar/ui/events/ens_events.cljs

```diff
@@ -103,3 +114,59 @@
   (fn [{:keys [:db]} [node]]
     (let [{:keys [:ens.record/owner]} (get-ens-record db node)]
       {:dispatch [:public-resolver.name/load owner]})))
+
+(reg-event-fx
+  :ens/set-resolver
+  [interceptors (validate-first-arg (s/keys :req [:ens.record/name]
+                                            :opt [:ens.record/resolver]))]
+  (fn [{:keys [:db]} [form-data]]
+    (let [public-resolver (get-in db [:smart-contracts :public-resolver :address])
+          form-data (assoc form-data
+                           :ens.record/node (namehash (str (:ens.record/name form-data)
+                                                           constants/registrar-root))
+                           :ens.record/resolver (get form-data :ens.record/resolver public-resolver))
+          full-name (str (:ens.record/name form-data) constants/registrar-root)]
+      {:dispatch [:district0x/make-transaction
+                  {:name (gstring/format "Setup resolver for %s" full-name)
+                   :contract-key :ens
+                   :contract-method :set-resolver
+                   :form-data (select-keys form-data [:ens.record/node :ens.record/resolver])
+                   :args-order [:ens.record/node :ens.record/resolver]
+                   :form-id (select-keys form-data [:ens.record/node])
+                   :tx-opts {:gas 100000 :gas-price default-gas-price}
+                   :result-href (path-with-query (path-for :route.user/manage-names)
+                                                 {:name (:ens.record/name form-data)})
```

Comment:
.eth is missing here. When I go to URL like http://localhost:3001/manage-names?name=ethlance it won't load ownership